pom.xml

```diff
@@ -94,6 +94,23 @@
         <artifactId>json-schema-validator</artifactId>
         <version>2.2.14</version>
       </dependency>
+      <dependency>
+        <!-- TODO: Remove once this version is included in BOM -->
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>script-security</artifactId>
+        <version>1172.v35f6a_0b_8207e</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins.workflow</groupId>
+        <artifactId>workflow-cps</artifactId>
+        <version>2699.v7231c3b_4a_231</version> <!-- TODO: https://github.com/jenkinsci/workflow-cps-plugin/pull/538 -->
```

Comment:
This override is just here temporarily right?

Reply:
I have it set to this version right now to test that everything works once the relevant entries are removed from `default-allowlist` upstream. Once this PR has been reviewed, I will change it to `2692.v76b_089ccd026` (first version with the new API) so it can be merged and released, unless that version of `workflow-cps` has made it into a BOM release in the meantime, in which case I will update the BOM and delete this.

Reply:
No BOM release yet, so I downgraded `workflow-cps` in [36347d3](https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/529/commits/36347d379ba56dc60bacb17e7c3f9f6ab4eadfbc).